docs/kb/semgrep-ci/collect-gha-logs.md

```diff
@@ -0,0 +1,15 @@
+# Collecting Semgrep GitHub Actions logs from Github
+
+1. On Github.com, or in your GitHub instance, navigate to the main page of the repository that Semgrep scanned.
+2. Click the **Actions** tab.      
+    ![actions-tab](https://docs.github.com/assets/cb-45938/images/help/repository/actions-tab.png)  
+3. In the left sidebar, click the GitHub Actions workflow you want to retrieve logs for. The name depends on your configuration but defaults to "Semgrep". In this example it is "Scan":
```

Comment:
```suggestion
3. In the left sidebar, click the GitHub Actions workflow you want to retrieve logs for. The name depends on your configuration but defaults to "Semgrep." In this example it is "Scan":
```
[GSG thing: Commas and periods go inside quotation marks](https://developers.google.com/style/quotation-marks#commas-and-periods-with-quotation-marks)